### ISSUE TITLE
After modifying the source file, the corresponding automatically generated class is not updated

### ISSUE THREAD

**Issue by changchengqin** [vert-x3/vertx-codegen#309]
### Questions
After modifying the source file, the corresponding automatically generated class is not updated.

### Version

4.0.0-milestone4

### Context

while I used vertx to automatically generate classes such as VertxProxyHandler and VertxEBProxy, I encountered a problem, that is:
when I modify a certain service source code file, the IDE will automatically recompile it, but the corresponding VertxProxyHandler and VertxEBProxy classes have not been recompile.

After debugging, I found that it was caused by this line of code in the process method of the CodeGenProcessor class in the vertx-codgen class library.

![image](https://user-images.githubusercontent.com/3336443/89724518-9cae9e00-da36-11ea-89ee-ad555ecfc7dc.png)

As long as classes such as VertxProxyHandler are automatically generated once, unless the entire project is recompiled as a whole or the generated class is deleted, it will not be automatically generated in the future.

### Do you have a reproducer?

https://github.com/changchengqin/codegen

### Steps to reproduce

1. clone above repo.
2. build
3. modify source file `src/main/java/com/example/codegen/service/UserService.java`
4. recompile source file `src/main/java/com/example/codegen/service/UserService.java`

The normal expected result is that UserServiceVertxProxyHandler and UserServiceEBProxy should be updated accordingly, but the fact is not.
### Extra

* OS: Windows 10 Enterprise
* JDK: 11 or later
* IDE: Intellij IDEA Community 2019.3.1 or 2019.3.3 or 2019.3.4 or 2020.2

**Comment by vietj** [comment]
have you tried modifying this project to make it work ?

**Comment by changchengqin** [comment]
Yes, I tried many times, including using the latest Vertx 4.0.0.beta3 version, the phenomenon still exists.

**Comment by vietj** [comment]
no I mean, have you tried patching vertx-codegen to make it work

**Comment by changchengqin** [comment]
Hmm, yes, when I commented these three lines of code in the process method of the CodeGenProcessor class, everything returned to normal.
![image](https://user-images.githubusercontent.com/3336443/97536515-3b281b00-19f8-11eb-96cc-399d2ffa6738.png)
